tokens/index.d.ts

```diff
@@ -0,0 +1,9 @@
+export type UserToken = {
+  id: string;
+  type: UserTokenType;
+  value: string;
+  createdAt: Date;
+  expiresAt: Date;
+}
+
+export type UserTokenType = "recovery" | "email" | "phone";
```

Comment:
add ``organization-invite`` token type ?

Reply:
It's already in the next update but in another branch (and authentication token type added too)